dhis-2/dhis-api/src/main/java/org/hisp/dhis/user/User.java

```diff
@@ -225,7 +225,7 @@
 
     private Date lastCheckedInterpretations;
 
-    private Set<UserGroup> groups = new HashSet<>();
+    private Set<UserGroup> userGroups = new HashSet<>();
```

Comment:
Is there any backwards API compatibility issues with this change? Or the JSON data model stays the same?

Reply:
JSON model should stay the same. This is just for code clarity. So the code matches the output model naming.